sql/2020/06_Third_Parties/percent_of_third_party_cache.sql

```diff
@@ -0,0 +1,59 @@
+#standardSQL
+# Percent of third party requests cached
+# Cache-Control documentation: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#Directives
+
+WITH requests AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    resp_cache_control,
+    status,
+    respOtherHeaders,
+    reqOtherHeaders,
+    type,
+    req_host AS host
+  FROM
+    `httparchive.summary_requests.2020_08_01_*`
+),
+third_party AS (
+  SELECT
+    domain
+  FROM
+    `httparchive.almanac.third_parties`
+  WHERE
+    date = '2020-08-01'
+),
+base AS (
+  SELECT
+    client,
+    type,
+    IF(
+    (
+      status IN (301, 302, 307, 308, 410)
+      AND NOT REGEXP_CONTAINS(resp_cache_control, r'private|no-store')
```

Comment:
For regexes like these, should they be case-insensitive?
```suggestion
      AND NOT REGEXP_CONTAINS(resp_cache_control, r'(?i)private|no-store')
```